### PR TITLE
add Qlik page, adjust Migration nav, tweak DMS docs

### PR DESCRIPTION
**Pull request by taroface**
DOC-6986

- Add a high-level page for Qlik Replicate.
- Update third-party tooling table.
- Adjust Migrate nav with third-party tooling options.
- Minor updates to DMS docs.

### CODE REVIEW

File: v23.1/qlik.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- [Migrate data to CockroachDB](#migrate-and-replicate-data-to-cockroachdb) from an existing, publicly hosted database containing application data, such as PostgreSQL, MySQL, Oracle, or Microsoft SQL Server. 
+- [Replicate data to a secondary source](#replicate-data-from-cockroachdb-to-a-secondary-source) such as Kafka or cloud storage.
+
+This page describes the Qlik Replicate functionality at a high level. For detailed information, refer to the tutorial and documentation provided when [signing up for Qlik Replicate](https://www.qlik.com/us/products/qlik-replicate).
+
+## Before you begin
+
+Complete the following items before using Qlik Replicate:
+
+- Ensure you have a secure, publicly available CockroachDB cluster running the latest **{{ page.version.version }}** [production release](../releases/index.html), and have created a [SQL user](security-reference/authorization.html#sql-users) that you can use for your Qlik Replicate target endpoint.
```
Comment (otan) [inline v23.1/qlik.md:19]: the copy_from_blah commands need to be changed as well.

**Review by otan — APPROVED** [review]
the info LGTM, but i'm lukewarm on exposing it (not up to me though!).

File: v23.1/qlik.md (hunk 1 above)
Comment (taroface) [reply v23.1/qlik.md:19]: Added the two session var workarounds for 22.2 - this is what you meant, right?

Comment (otan) [reply v23.1/qlik.md:19]: yes, they probs need to be in 23.1 too

Comment (taroface) [reply v23.1/qlik.md:19]: Ah right, added.

**Review by kathancox — APPROVED** [review]
A couple of non-blocking comments Ryan, but LGTM!

File: v22.2/qlik.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+This page describes the Qlik Replicate functionality at a high level. For detailed information, refer to the tutorial and documentation provided when [signing up for Qlik Replicate](https://www.qlik.com/us/products/qlik-replicate).
+
+## Before you begin
+
+Complete the following items before using Qlik Replicate:
+
+- Ensure you have a secure, publicly available CockroachDB cluster running the latest **{{ page.version.version }}** [production release](../releases/index.html), and have created a [SQL user](security-reference/authorization.html#sql-users) that you can use for your Qlik Replicate target endpoint.
+    - If your CockroachDB cluster is running v22.2.4 or later, set the following session variables using [`ALTER ROLE ... SET {session variable}`](alter-role.html#set-default-session-variable-values-for-a-role):
+
+        {% include_cached copy-clipboard.html %}
```
[the remark this answers is not in the document]
Comment (taroface) [reply v22.2/qlik.md:22]: We do need a dedicated session variables reference. We usually link to `set-vars.html#supported-variables` which is what I've done now.

File: v22.2/qlik.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        ~~~
+
+        {% include_cached copy-clipboard.html %}
+        ~~~ sql
+        ALTER ROLE {username} SET copy_from_atomic_enabled = false;
+        ~~~
+
+        This prevents a potential issue when migrating especially large tables with millions of rows.
+
+- If you are migrating to a {{ site.data.products.db }} cluster and plan to [use replication as part of your migration strategy](#migrate-and-replicate-data-to-cockroachdb), you must first **disable** [revision history for cluster backups](take-backups-with-revision-history-and-restore-from-a-point-in-time.html).
```
[the remark this answers is not in the document]
Comment (taroface) [reply v22.2/qlik.md:34]: It's complicated, but the migration won't succeed unless you do so. Added a statement to this effect :)

File: v22.2/qlik.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    {{site.data.alerts.callout_danger}}
+    You will not be able to run a [point-in-time restore](take-backups-with-revision-history-and-restore-from-a-point-in-time.html#point-in-time-restore) as long as revision history for cluster backups is disabled. Once you verify in Monitor view that the migration succeeded, you should re-enable revision history.
```
[the remark this answers is not in the document]
Comment (taroface) [reply v22.2/qlik.md:36]: Qlik Replicate! Clarified.

File: v22.2/qlik.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+- Manually create all schema objects in the target CockroachDB cluster. This is required in order for Qlik Replicate to populate data successfully.
+    - If you are migrating from PostgreSQL, MySQL, Oracle, or Microsoft SQL Server, [use the **Schema Conversion Tool**](../cockroachcloud/migrations-page.html) to convert and export your schema. Ensure that any schema changes are also reflected on your tables, or add transformation rules. If you make substantial schema changes, the Qlik Replicate migration may fail.
+
+    {{site.data.alerts.callout_info}}
+    All tables must have an explicitly defined primary key. For more guidance, see [Migrate Your Database to CockroachDB](migration-overview.html#step-1-convert-your-schema).
+    {{site.data.alerts.end}}
+
+## Migrate and replicate data to CockroachDB
+
+You can use Qlik Replicate to migrate tables from a source database to CockroachDB. This can comprise an initial load that copies the selected schemas and their data from the source database to CockroachDB, followed by continuous replication of ongoing changes using change data capture (CDC).
```
[the remark this answers is not in the document]
Comment (taroface) [reply v22.2/qlik.md:50]: Done.

**Pull request merged**